src/main/java/gregtech/integration/forestry/recipes/MiscRecipes.java

```diff
@@ -0,0 +1,442 @@
+                .duration(64).EUt(16).buildAndRegister();
+
+        RecipeMaps.MIXER_RECIPES.recipeBuilder()
+                .input(MetaItems.FERTILIZER)
+                .input(Blocks.DIRT, 8, true)
+                .fluidInputs(Materials.Water.getFluid(1000))
+                .outputs(GTUtility.copy(8, humus))
+                .duration(64).EUt(16).buildAndRegister();
+
+        // Cans, Capsules, etc
+        RecipeMaps.ASSEMBLER_RECIPES.recipeBuilder()
+                .input(OrePrefix.plate, Materials.Tin, 2)
+                .input(Blocks.GLASS_PANE)
+                .circuitMeta(1)
+                .outputs(IntegrationUtil.getModItem(GTValues.MODID_FR, "can", 0))
+                .duration(120).EUt(7).buildAndRegister();
+
+        RecipeMaps.EXTRUDER_RECIPES.recipeBuilder()
+                .inputs(IntegrationUtil.getModItem(GTValues.MODID_FR, "beeswax", 0))
+                .notConsumable(MetaItems.SHAPE_EXTRUDER_CELL)
+                .outputs(IntegrationUtil.getModItem(GTValues.MODID_FR, "capsule", 0))
+                .duration(64).EUt(16).buildAndRegister();
+
+        RecipeMaps.EXTRUDER_RECIPES.recipeBuilder()
+                .inputs(IntegrationUtil.getModItem(GTValues.MODID_FR, "refractory_wax", 0))
+                .notConsumable(MetaItems.SHAPE_EXTRUDER_CELL)
+                .outputs(IntegrationUtil.getModItem(GTValues.MODID_FR, "refractory", 0))
+                .duration(128).EUt(16).buildAndRegister();
+
+        // Propolis
+        RecipeMaps.CENTRIFUGE_RECIPES.recipeBuilder()
+                .inputs(IntegrationUtil.getModItem(GTValues.MODID_FR, "propolis", 0))
+                .output(MetaItems.STICKY_RESIN)
+                .duration(128).EUt(5).buildAndRegister();
+
+        if (Loader.isModLoaded(GTValues.MODID_GENETICS)) {
+
+            // DNA Dye
+            RecipeMaps.MIXER_RECIPES.recipeBuilder()
+                    .input(OrePrefix.dust, Materials.Glowstone)
+                    .input("dyePurple")
+                    .input("dyeBlue")
+                    .outputs(IntegrationUtil.getModItem(GTValues.MODID_GENETICS, "misc", 1, 8))
+                    .duration(100).EUt(GTValues.VA[GTValues.LV]).buildAndRegister();
+
+            // Fluorescent Dye
+            RecipeMaps.MIXER_RECIPES.recipeBuilder()
+                    .input(OrePrefix.dust, Materials.Glowstone)
+                    .input("dyeOrange")
+                    .input("dyeYellow")
+                    .outputs(IntegrationUtil.getModItem(GTValues.MODID_GENETICS, "misc", 2, 2))
+                    .duration(100).EUt(GTValues.VA[GTValues.LV]).buildAndRegister();
+
+            // Growth Medium
+            RecipeMaps.MIXER_RECIPES.recipeBuilder()
+                    .input(OrePrefix.dust, Materials.Sugar)
+                    .input(OrePrefix.dust, Materials.Bone)
+                    .outputs(IntegrationUtil.getModItem(GTValues.MODID_GENETICS, "misc", 4, 2))
+                    .duration(400).EUt(16).buildAndRegister();
```

Comment:
I held off since I didn't want to make these recipes substantially harder than the vanilla recipes. Maybe if we add a more extensive harder recipe config we can do something like that